method.h

```diff
@@ -177,6 +177,7 @@ struct rb_method_definition_struct {
     } body;
 
     ID original_id;
+    uintptr_t created_at;
```

Comment:
* why not `rb_serial_t`?
* `created_at` seems time, but it should be id (serial number).

Reply:
@ko1 `rb_serial_t` can be _bigger_ than a pointer.  On a 32bit system `uintptr_t` is 32bit while `rb_serial_t` is 64bit.  On such system using `rb_serial_t` here could change arrangement of members of `struct rb_call_cache`.  So I chose `uintptr_t`.

Reply:
Re: `created_at` seems time -- Yes it does.  The value of this field is a monotonic increasing integer unique to each definition.  It is much like Subversion's revisions.  So it is a serial number, and at the same time, a timestamp (though not proportional to wallc lock).

Reply:
ok. 32bit serial counter can be overflow so that rb_serial_t is introduced.